src/pudl/transform/classes.py

```diff
@@ -552,7 +552,7 @@ def convert_units(col: pd.Series, params: UnitConversion) -> pd.Series:
         new_name = col.name
     if (col.name == new_name) & (params.from_unit != "") & (params.to_unit != ""):
         logger.debug(f"Old and new column names are identical: {col.name}.")
-    col = (params.multiplier * col) + params.adder
+    col = (params.multiplier * pd.to_numeric(col)) + params.adder
```

Comment:
There were half a dozen tables where `col` ended up being a numeric string like `"0.0"` as a result of `ferc-xbrl-extractor==1.1.1` -- though I'm not sure what about the change in method over there affected the output type.

Reply:
There are notes in the PRs / issues but the thing that changed is what kind of NULL value ends up being generated. In v1.1.0 with `gb.ffill().iloc[-1]` we ended up with mixed `str` + `np.nan` => `object` columns that resulted in numeric values.  In v1.1.1 using `gb.last()` we ended up with mixed `str` + `None` => `object` columns that are interpreted as strings. 🤷🏼